src/test/java/com/cdancy/bitbucket/rest/features/BranchApiLiveTest.java

```diff
@@ -136,12 +140,49 @@ public void testDeleteBranchPermission() {
         }
     }
 
+    @Test(dependsOnMethods = {"testCreateBranch", "testListBranches"})
+    public void testGetBranchModelConfiguration() {
+        branchModelConfiguration = api().getModelConfiguration(projectKey, repoKey);
+        checkDefaultBranchConfiguration();
```

Comment:
Seen you pushed a commit but missed this. Not sure if you were planning on addressing with a subsequent commit.